signer/Config.go

```diff
@@ -3,44 +3,28 @@ package signer
 import (
 	"fmt"
 	"os"
-
-	"github.com/BurntSushi/toml"
 )
 
 type NodeConfig struct {
-	Address string `toml:"address"`
+	Address string
 }
 
 type CosignerConfig struct {
-	ID          int    `toml:"id"`
-	Address     string `toml:"remote_address"`
-	RaftAddress string `toml:"remote_raft_address"`
+	ID          int
+	Address     string
+	RaftAddress string
 }
 
 type Config struct {
-	Mode              string           `toml:"mode"`
-	PrivValKeyFile    string           `toml:"key_file"`
-	PrivValStateDir   string           `toml:"state_dir"`
-	ChainID           string           `toml:"chain_id"`
-	CosignerThreshold int              `toml:"cosigner_threshold"`
-	ListenAddress     string           `toml:"cosigner_listen_address"`
-	RaftListenAddress string           `toml:"cosigner_raft_listen_address"`
-	Nodes             []NodeConfig     `toml:"node"`
-	Cosigners         []CosignerConfig `toml:"cosigner"`
-}
-
-func LoadConfigFromFile(file string) (Config, error) {
```

Comment:
is this no longer used?

Reply:
no usages in the project. wasn't a recent change from what I can tell, just dead code that's been in here a while.